packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -1057,13 +1141,15 @@ to access the OIDC `/userinfo` [endpoint](/docs/reference/api/oidc/#userinfo). T
 | address          | Requests access to the `address` claim.                                                                         | No             |
 | groups           | Requests access to the `groups` claim.                                                                          | No             |
 | offline_access   | Requests a refresh token used to obtain more access tokens without re-prompting the user for authentication.   | No             |
+| device_sso <ApiLifecycle access="ea" />   | Requests a device secret used to obtain a new set of tokens without re-prompting the user for authentication. See *link*  | No             |
 
 ### Scope values
 
 * `openid` is required for any OpenID request connect flow. If the `openid` scope value isn't present, the request may be a valid OAuth 2.0 request, but it's not an OpenID Connect request.
 * `profile` requests access to these default profile claims: `name`, `family_name`, `given_name`, `middle_name`, `nickname`, `preferred_username`, `profile`, `picture`, `website`, `gender`, `birthdate`, `zoneinfo`,`locale`, and `updated_at`.
 * `offline_access` can only be requested in combination with a `response_type` that contains `code`. If the `response_type` doesn't contain `code`, `offline_access` is ignored.
 * For more information about `offline_access`, see the [OIDC spec](http://openid.net/specs/openid-connect-core-1_0.html#OfflineAccess).
+* For more information about `device_sso`<ApiLifecycle access="ea" />, see *link*.
```

Comment:
link?

Reply:
@susanharper-okta to substitute when the guide has a link

Reply:
See [Configure Device Authorization](/docs/guides/device-authorization-grant/main/)

Reply:
That's the guide for device flow, this feature is Native SSO, do we have a guide link for that ready?